common/pulp/common/constants.py

```diff
@@ -64,3 +64,6 @@
 DEFAULT_CA_PATH = '/etc/pki/tls/certs/ca-bundle.crt'
 
 SCHEDULER_WORKER_NAME = "scheduler"
+# CeleryBeat HA constants
```

Comment:
This comment could be generalized some. Consider moving this comment above the line above and rewording it to be `# celerybeat constants`